clar2wasm/tests/wasm-generation/blockinfo.rs

```diff
@@ -62,3 +66,37 @@ proptest! {
         }
     }
 }
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crosscheck_at_block(
+        value in PropValue::any(),
+        buf in buffer(32)
+    ) {
+        crosscheck(
+            &format!("(at-block {buf} {value})"),
+            Ok(Some(value.into()))
+        )
+    }
+
+    #[test]
+    fn crosscheck_at_block_after_scope(
+        value in PropValue::any(),
+        buf in buffer(32)
+    ) {
+        let expected_block = Value::UInt(0);
+
+        let crosscheck_for = |epoch: StacksEpochId, expected: Value, keyword: &str| {
+            crosscheck_with_epoch(
+                &format!("(at-block {buf} {value}) (ok {keyword})"),
+                Ok(Some(Value::okay(expected).unwrap())),
+                epoch,
+            );
+        };
+
+        crosscheck_for(StacksEpochId::Epoch30, expected_block.clone(), "stacks-block-height");
```

Comment:
According to the [docs](https://docs.stacks.co/reference/keywords#stacks-block-height), `stacks-block-height` is only available for clarity v3. But the v2 test doesn't fail, as `crosscheck_with_epoch` bypasses the test config (as expected, I guess?).

Should we change the prop tests using `crosscheck_with_epoch` in favor of the new feature gates? If so, can you open an about it (no need to address it in this PR)

Reply:
Created an issue about that. https://github.com/stacks-network/clarity-wasm/issues/518